apps/services/auth/ids-api/src/app/delegations/delegations.controller.spec.ts

```diff
@@ -79,11 +70,28 @@ describe('DelegationsController', () => {
     let delegationProviderModel: typeof DelegationProviderModel
     let delegationIndexService: DelegationsIndexService
 
+    const client = createClient({
+      clientId: '@island.is/webapp',
+    })
+
+    client.allowedScopes = Object.values([
+      'scope/valid1',
+      'scope/valid2',
+      'scope/valid1and2',
+      'scope/unactiveType',
+      'scope/outdated',
+      'disabledScope',
```

Comment:
Should these not be in variables so they do not have to be referenced as strings in the tests?

Reply:
Fixed.